src/LicenseChecker.php

```diff
@@ -65,43 +76,42 @@ protected function execute(InputInterface $input, OutputInterface $output): int
             return self::FAILURE;
         }
 
-        $process = Process::fromShellCommandline('SHELL_VERBOSITY=0 composer licenses --format=json', $path);
-        $process->run();
-        if (!$process->isSuccessful()) {
-            $style->error(\sprintf('Failed to run "composer licenses --format=json" (%d).', $process->getExitCode()));
+        $path = (string) \realpath($path ?? \dirname(__DIR__));
+        /** @var string $providerId */
```

Comment:
```suggestion
        /** @var non-empty-string $providerId */
```

Though I think we should just check this at runtime fully instead.

Reply:
This will fail while trying to locate, guess that's fine?